nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-resources/src/main/resources/bin/nifi.sh

```diff
@@ -333,7 +333,20 @@ run() {
     sleep 3
     echo
 }
+stateless(){
```

Comment:
@SamHjelmfelt  what do you think about adding 'Use of this command is considered experimental and the commands and approach used may change from time to time.